common/build.gradle

```diff
@@ -10,27 +10,21 @@ plugins {
     id 'java-library'
 }
 
-repositories {
-    // Use Maven Central for resolving dependencies.
-    mavenCentral()
```

Comment:
This one also needs snapshot/milestone (taken care of now that its moved to root project)